content/docs/resiliency/_index.md

```diff
@@ -189,6 +189,9 @@ Similarly, the label selector for csi-powerscale, csi-unity, csi-powerstore and
 
  3. As noted previously in the Limitations and Exclusions section, CSM for Resiliency has not yet been verified to work with ReadWriteMany or ReadOnlyMany volumes. Also, it has not been verified to work with pod controllers other than StatefulSet.
 
+ ### Storage Array Upgrades
+To avoid application pods getting stuck in a Pending state, CSM for Resiliency should be disabled for storage array upgrades; even if the storage array upgrade is advertised as non-distruptive. If the container orchestrator platform nodes lose connectivity with the array, which is more likely during an upgrade, then Resiliency will delete the application pods on the affected nodes and attempt to move them to a healthy node. If all of the nodes are affected, then the application pods will be stuck in a Pending state.
```

Comment:
Is there another place where we could also reference this as a prerequisite? Chances are it will not be noticed in this page, it needs to be a reminder before any storage upgrade.

Reply:
This section is now added to the Deployment documentation for Helm and Operator as Prerequisite.